price_pusher/package.json

```diff
@@ -43,6 +43,7 @@
     "typescript": "^4.6.3"
```

Comment:
I think we need to bump version as major for this change.

Reply:
will send a pr with readme and version changes. which will be the final one after refactoring.